frontend/src/pages/Cct/index.jsx

```diff
@@ -0,0 +1,195 @@
+import { useEffect, useRef, useState } from "react";
+import { MovableDiv, PageTitle, WidgetsModal } from "../../components"
+import { useLocation, useNavigate, useParams } from "react-router";
+import { IoMdArrowUp, IoMdArrowDown } from "react-icons/io";
+import templateservice from "../../services/templateService";
+import widgetService from "../../services/widgetService";
+import { motion } from "framer-motion";
+import { FiTrash2 } from "react-icons/fi";
+
+const colors = [
+    'bg-pink-200 border-pink-400',
+    'bg-rose-200 border-rose-400',
+    'bg-violet-200 border-violet-400',
+    'bg-blue-200 border-blue-400',
+    'bg-green-200 border-green-400',
+    'bg-yellow-200 border-yellow-400',
+    'bg-orange-200 border-orange-400',
+    'bg-stone-200 border-stone-400',
+    'bg-deep-purple-200 border-deep-purple-400'
+];
```

Comment:
You could move this to root so that it only loads once throughout the entire application (if you do that, then remove it from all the pages that use it)

Reply:
that doest work